baseapp/baseapp.go

```diff
@@ -24,6 +24,8 @@ import (
 const (
 	// Check a transaction
 	runTxModeCheck runTxMode = iota
+	// Recheck a (pending) transaction after a commit
+	runTxModeReCheck runTxMode = iota
```

Comment:
there's no need to redeclare `iota` here. Ditto on the rest of the `runTxMode`s.